bouncer/scripts/redirect.js

```diff
@@ -81,6 +81,11 @@ export async function redirect(
     return;
   }
 
+  if (settings.alwaysUseVia) {
+    navigateTo(settings.viaUrl);
+    return;
+  }
```

Comment:
We could cut out the frontend in this case and just issue a 3xx redirect from the backend, but this approach makes the flow more similar across the different cases.